FolfApp/src/components/courseDetails.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import {gql, graphql} from 'react-apollo';
+import {
+  comp as CourseListRe,
+} from '../../lib/js/src/components/courseDetails.js';
+
+class CourseDetail extends React.Component {
+  static navigationOptions = {
+    title: 'Course Details',
+  };
+  render() {
+    return (
+      <CourseListRe
+        course={this.props.data && this.props.data.course}
+        loading={this.props.data.loading}
```

Comment:
Why  course={this.props.data && this.props.data.course}
and not   loading={this.props.data && this.props.data.loading} ?